build/fix-api-docs.js

```diff
@@ -0,0 +1,20 @@
+var replace = require("replace");
+var path = require('path')
+var apiPath = path.join(__dirname, '..', 'docs', 'api');
+
+var replacements = [
+  {find: '\"/docs/guides/\"', replace: '"#"'},
+  {find: '\"/docs/guides/(.*)\.md\"', replace: '"tutorial-$1.html"'},
+  {find: '\"tutorial-tech.html\"', replace: '"tutorial-tech_.html"'}
```

Comment:
I'm not entirely sure, I think it may have something to do with a file name tech already existing? Either that or the source handler weirdness, (jsdoc links to `-_Tech.html` on the Tech API docs)